contributing/improve-website.md

```diff
@@ -33,7 +33,7 @@ git checkout asf-site
 
 ## Directory structure and files
 
-Flink's website is written in [Markdown](http://daringfireball.net/projects/markdown/). Markdown is a lightweight markup language which can be translated to HTML. We use [Jekyll](http://jekyllrb.com/) to generate static HTML files from Markdown.
+Flink's website is written in [Markdown](http://daringfireball.net/projects/markdown/). Markdown is a lightweight markup language which can be translated to HTML. We use [Hugo](https://gohugo.io/documentation/) to generate static HTML files from Markdown.
```

Comment:
The website build didn't change — it's still Jekyll-based. Only the Flink docs build was migrated to Hugo!